service/terraform-plugin/src/main/java/com/solace/maas/ep/event/management/agent/plugin/terraform/manager/TerraformManager.java

```diff
@@ -7,6 +7,7 @@
 import com.solace.maas.ep.event.management.agent.plugin.command.model.CommandResult;
 import com.solace.maas.ep.event.management.agent.plugin.command.model.JobStatus;
 import com.solace.maas.ep.event.management.agent.plugin.constants.RouteConstants;
+import com.solace.maas.ep.event.management.agent.plugin.messagingService.event.EventProperty;
```

Comment:
I guess this library doesn't have PMD checks. Otherwise, it should have failed the build for un-used imports. 

Please just roll-back this file considering there is no relevant change here.